src/routes/teacher/ClassSettings.tsx

```diff
@@ -19,7 +19,7 @@ export default function ClassSettings({
 }) {
 	return (
 		<>
-			<Grid item xs={12}>
+			{/* <Grid item xs={12}>
```

Comment:
I appreciate that you comment out code that we might need later, but best practice is actually to just delete any commented code that you're only thinking of saving for later. Comments aren't updated with the code around them, so they quickly become deprecated as the rest of the codebase develops. Consider code to be deprecated the moment you decide to comment it out, and delete it instead (we have git history so we can always retrieve it anyway!)